search/search-domain/src/main/java/io/camunda/search/filter/DecisionInstanceFilter.java

```diff
@@ -8,23 +8,26 @@
 package io.camunda.search.filter;
 
 import static io.camunda.util.CollectionUtil.addValuesToList;
+import static io.camunda.util.CollectionUtil.collectValues;
 import static io.camunda.util.CollectionUtil.collectValuesAsList;
 
 import io.camunda.search.entities.DecisionInstanceEntity.DecisionDefinitionType;
 import io.camunda.search.entities.DecisionInstanceEntity.DecisionInstanceState;
+import io.camunda.util.FilterUtil;
 import io.camunda.util.ObjectBuilder;
+import java.time.OffsetDateTime;
 import java.util.Collections;
 import java.util.List;
 import java.util.Objects;
 
 public record DecisionInstanceFilter(
```

Comment:
💭 I was just wondering: should we update all attributes to be "operation-ready" already? That makes it a lot easier in the future to simply add Advanced Search capabilities to all attributes (can even be done step by step for chunks of attributes in the REST request mapper).

Reply:
That is also what we did for the process instance filter.

Reply:
We could do it, it's not a huge effort anymore as we have all the search operation query builder methods ready.
Are there any fields where operations don't make sense? Or can I add it for _all_ of them?

Reply:
I think only `boolean` fields in general don't get advanced search, I think.